next.config.js

```diff
@@ -27,6 +27,11 @@ module.exports = {
     return config;
   },
   redirects: async () => [
+    {
+      source: '/',
+      destination: '/buolas',
+      permanent: false,
```

Comment:
true is very annoying especially on localhost especially when the source is '/' 😛 my localhost in mono repo is always like this `http://localhost:4200/ethereum/components`

Reply:
i don't think `/` should be a permanent redirect :/ it could change in the future, if an index landing page is implemented

would generally keep permanent redirects for when a page _moves_ to a new permalink, i.e. a blog-post changes it's slug or some static page changes location